code/datums/webhook.dm

```diff
@@ -0,0 +1,71 @@
+var/list/global_webhooks = list()
+
+/hook/startup/proc/initialize_webhooks()
+	return global_initialize_webhooks()
```

Comment:
Might be better to stick this in early/late misc init instead of a global hook, though I suppose it doesn't really matter.